web/src/components/countrytable.js

```diff
@@ -39,12 +41,15 @@ const RECT_OPACITY = 0.8;
 const X_AXIS_HEIGHT = 15;
 const SCALE_TICKS = 4;
 
-const getProductionData = data => modeOrder.map((mode) => {
+const getProductionData = (data, timescale) => modeOrder.map((mode) => {
   const isStorage = mode.indexOf('storage') !== -1;
   const resource = mode.replace(' storage', '');
 
   // Power in MW
-  const capacity = (data.capacity || {})[mode];
+  // Do not show capacity if timescale != live
```

Comment:
Interesting.. Maybe this is not such an obvious choice (the alternative being showing the monthly averages for capacities), but I agree it's better to remove it than risk wrong interpretation of the capacity value being lower than production :slightly_smiling_face: